cli/schema.go

```diff
@@ -0,0 +1,101 @@
+package cli
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"time"
+
+	"github.com/cloudspannerecosystem/harbourbridge/conversion"
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/google/subcommands"
+)
+
+// SchemaCmd struct with flags.
+type SchemaCmd struct {
+	prompt           bool
+	filePrefix       string
+	driverName       string
+	schemaSampleSize int64
+	dumpFilePath     string
+	targetDb         string
+}
+
+// Name returns the name of operation.
+func (cmd *SchemaCmd) Name() string {
+	return "schema"
+}
+
+// Synopsis returns summary of operation.
+func (cmd *SchemaCmd) Synopsis() string {
+	return "generate schema for target db from source db schema"
+}
+
+// Usage returns usage info of the command.
+func (cmd *SchemaCmd) Usage() string {
+	return fmt.Sprintf(`%v schema [file]...
+
+Convert schema for source db specified by driver. Source db dump files can be
+specified by either dump-file flag or from stdin. Connection profile for source
```

Comment:
nit: Should we say "source db dump **data**" or some other better phrasing as we end the sentence with "from stdin"? I know we use dumpfiles realistically in stdin but this way we stay consistent with what we actually implement in harbourbridge i.e. reading data from stdin.

Reply:
Even from stdin, we read dump file only. Dump file itself can contain either/or/both of schema and data statements. I have though rephrased the statement now as "Source db dump file can be specified by either dump-file flag or piped to stdin". Done @https://github.com/cloudspannerecosystem/harbourbridge/pull/175/commits/be4ddfc12ff3c3c86abfdaa613cfa9f0d1fc2a12. WDYT?

Reply:
This is the best rephrasing according to me!